ironfish/src/webApi.ts

```diff
@@ -27,8 +27,10 @@ export type ApiMaspUpload = {
   }
   transactions: {
     hash: string
-    type: MaspTransactionTypes
-    assetName: string
```

Comment:
This update is a result of updating the api here:
https://github.com/iron-fish/ironfish-api/pull/1156
https://github.com/iron-fish/ironfish-api/pull/1157